### PR TITLE
Update recipe

### PR DESCRIPTION
**Pull request by ericpre**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ericpre** [comment]
@conda-forge-admin, please rerender

**Comment by ericpre** [comment]
@CJ-Wright, this is ready for the review!

**Comment by thewchan** [comment]
Any news on this PR? Got a package waiting for the newest version

**Comment by thewchan** [comment]
It looks like all tests are passing except for the linter? @conda-forge/core

**Comment by tacaswell** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,47 +1,52 @@
...
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true   # [py2k]
   entry_points:
     - fonttools = fontTools.__main__:main
     - ttx = fontTools.ttx:main
     - pyftsubset = fontTools.subset:main
     - pyftmerge = fontTools.merge:main
-    - pyftinspect = fontTools.inspect:main
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (tacaswell) [inline recipe/meta.yaml:21]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

**Comment by tacaswell** [comment]
I can not make any sense of the linter's complaint.  I'm opening a new PR to see if it will behave any betterr...

**Comment by tacaswell** [comment]
The bot in #77 removed the newline in meta.yaml so even though the file in this PR looks ok, it is failed linting when merged.

**Comment by tacaswell** [comment]
@thewchan Yes, this can be closed.  

Thanks @ericpre and @dopplershift for getting this moving.

**Pull request closed without merging**